specification/monitor/resource-manager/readme.md

```diff
@@ -35,13 +35,55 @@ title: MonitorManagementClient
 ``` yaml
 description: Monitor Management Client
 openapi-type: arm
-tag: package-2021-09
+tag: package-2022-02
 
 directive:
   - suppress: Example Validations
     reason: There are open issues (bugs) in the validator affecting some of the examples and since there is no way to selectively disable the validation for a particular example or paths, all of the example validation is being turned off.
 ```
 
+### Tag: package-2022-02
+
+These settings apply only when `--tag=package-2022-02` is specified on the command line
+
+``` yaml $(tag) == 'package-2022-02'
+input-file:
+- Microsoft.Insights/stable/2015-04-01/autoscale_API.json
+- Microsoft.Insights/stable/2015-04-01/operations_API.json
+- Microsoft.Insights/stable/2016-03-01/alertRulesIncidents_API.json
+- Microsoft.Insights/stable/2016-03-01/alertRules_API.json
+- Microsoft.Insights/stable/2016-03-01/logProfiles_API.json
+- Microsoft.Insights/preview/2017-05-01-preview/diagnosticsSettings_API.json
+- Microsoft.Insights/preview/2017-05-01-preview/diagnosticsSettingsCategories_API.json
+- Microsoft.Insights/stable/2021-09-01/actionGroups_API.json
+- Microsoft.Insights/stable/2015-04-01/activityLogs_API.json
+- Microsoft.Insights/stable/2015-04-01/eventCategories_API.json
+- Microsoft.Insights/stable/2015-04-01/tenantActivityLogs_API.json
+- Microsoft.Insights/stable/2018-01-01/metricDefinitions_API.json
+- Microsoft.Insights/stable/2018-01-01/metrics_API.json
+- Microsoft.Insights/stable/2019-03-01/metricBaselines_API.json
+- Microsoft.Insights/stable/2018-03-01/metricAlert_API.json
+- Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json
+- Microsoft.Insights/preview/2017-12-01-preview/metricNamespaces_API.json
+- Microsoft.Insights/preview/2018-11-27-preview/vmInsightsOnboarding_API.json
+- Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json
+- Microsoft.Insights/stable/2020-10-01/activityLogAlerts_API.json
+- Microsoft.Insights/preview/2021-09-01-preview/dataCollectionEndpoints_API.json
+- Microsoft.Insights/preview/2021-09-01-preview/dataCollectionRuleAssociations_API.json
+- Microsoft.Insights/preview/2021-09-01-preview/dataCollectionRules_API.json
+```
```

Comment:
Could we double confirm if the swaggers here are all in their latest api-version?
We have quite a few complains in the SDKs, such as this: https://github.com/Azure/azure-sdk-for-go/issues/16192 , since it turns out some old api-versions are no longer supported by the service.
This is optional, I am bringing this up as a FYI. If you do not want to do this in this PR, please write an email to the corresponding feature owners in the email and cc me so that we could track this issue.

Reply:
Hi @ArcturusZhang, I tried to update other RP's specs versions, but the problem is that those new api-versions are incompatible with old specs and the generations that are based on modeler v1 are broken.
I tried to apply some directives but with no luck, the model collisions cannot be solved by removing one of them.

Here the red box specs (old) are incompatible with the blue specs (new)
![image](https://user-images.githubusercontent.com/69991673/155242360-7c1987ac-0149-4825-b755-765998b6d63b.png)
 
With that definition .net & go pipelines are broken:
<img width="269" alt="image" src="https://user-images.githubusercontent.com/69991673/155242454-c555abf4-a8b6-4abb-a822-ccd67675e1b7.png">

I think the only way to make it work will be to migrate the pipelines to the newer modeler version.
I remember that there was an effort for this one, do you know if there's an ETA for it?